Bottom: 9925d30d928db61804ab3ae6d21d1811b1fde52c
Top:    91629b403e1dfaff9987390fd5f1a35964a9156a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:06 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 27e7c54..9f4b0e0 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1252,8 +1252,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
